### DISCUSSION TITLE
WindowsCommands

### DISCUSSION THREAD

**Discussion by mxsoftware** [AvaloniaUI/Avalonia#6451] [General]
In Avalonia is planned the introduction of WindowsCommands that behave like those introduced in the Mahapps library?

**Comment by maxkatz6** [comment]
No. 
From my understanding it can be easily replaced with https://github.com/wieslawsoltes/AvaloniaBehaviors or developers code.

**Comment by mxsoftware** [comment]
Thanks for the kind reply, but I don't think I have asked the question correctly.
Mahapps allows to insert icons in the window title, both on the left and on the right side, and associate commands.
I wanted to ask the Avalonia development team if they are planning to implement such a feature.
